jobs/JEVS_AQM_PLOTS

```diff
@@ -60,7 +60,7 @@ export mod_ver=${mod_ver:-${aqm_ver}}
 export VDATE=${VDATE:-${PDYm3}}
 
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver}/)}
-export EVSINaqm=${EVSINaqm:-$(compath.py ${envir}/com/${NET}/${evs_ver}/stats/${COMPONENT}/${MODELNAME})}
+export EVSINaqm=${EVSINaqm:-${COMIN}/stats/${COMPONENT}/${MODELNAME})
```

Comment:
@PerryShafran-NOAA Do you have a ) at the end of this new line instead of a }?

Reply:
export EVSINaqm=${EVSINaqm:-${COMIN}/stats/${COMPONENT}/${MODELNAME}}

Reply:
Dang, you have younger eyes than me!  They look almost the same when bunched up together!

I fixed it.